android-components/components/feature/prompts/src/main/java/mozilla/components/feature/prompts/concept/SimplePromptView.kt

```diff
@@ -0,0 +1,34 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.feature.prompts.concept
+
+/**
+ * An interface for views that can display a simple prompt.
```

Comment:
This interface doesn't seem as generic as the name implies? The `show` func requires a generatedPassword String, similar to the `onUseGeneratedPassword` func. Perhaps this could be something like `PasswordPromptView`?